.github/workflows/check.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     # https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
     strategy:
       matrix:
-        msrv: [1.65.0] # let .. else .. is stable in 1.65.0
```

Comment:
This is a big jump, what prompted it? The comment should be up-to-date on why this MSRV is what it is.

Reply:
I had this issue https://github.com/rust-lang/cargo/issues/10788
It got fixed at 1.71.0, I'll update the comment